test/mkdir.js

```diff
@@ -145,6 +159,21 @@ test('-p flag', t => {
   shell.rm('-Rf', `${t.context.tmp}/a`); // revert
 });
 
+test('create same dir twice with -p flag', t => {
```

Comment:
This is implicitly covered in the "globbed dir" test, but this merits explicit coverage.